controllers/apps/transformer_cluster_component.go

```diff
@@ -338,8 +345,8 @@ func newOrderedHandler(compSpecs map[string]*appsv1alpha1.ClusterComponentSpec,
 				orders: orders,
 			},
 			compPhasePrecondition: compPhasePrecondition{
-				orders:         orders,
-				expectedPhases: []appsv1alpha1.ClusterComponentPhase{appsv1alpha1.RunningClusterCompPhase},
+				orders:           orders,
+				phaseExpectation: upworking,
```

Comment:
@wangyelei Please make all ops follow the same orders in cluster topology, right now, they are all following orders in cluster, and their behaviors are in random and unfefined.